File: 2020-10-30-client-libraries/go/authentication.go
package main

import (
  "fmt"
  // "github.com/stripe/stripe-go/client"
  "github.com/stripe/stripe-go"
  "github.com/stripe/stripe-go/customer"
)

func main() {
  // Globally set API key
  // stripe.Key = "sk_test..."
  // c, _ := customer.Get("cus_ICiTLOiirpfd6d", nil)

  // Set API key per request
  // sc := &client.API{}
  // sc.Init("sk_test...", nil)
  // c, _ := sc.Customers.Get("cus_ICiTLOiirpfd6d", nil)
  // fmt.Println(c)

  // With connect:
  stripe.Key = "sk_test..."
  params := &stripe.CustomerParams{}
  params.SetStripeAccount("acct_1Ey3h1BqeQ4DKpna")
  c, _ := customer.Get("cus_HDfWzCQ6UEVtfu", params)
  fmt.Println(c)
}

File: 2020-10-30-client-libraries/php/authentication.php
<?php
require_once('vendor/autoload.php');

// Static methods
// Globally set API key
// \Stripe\Stripe::setApiKey('sk_test...');
//
// echo \Stripe\Customer::retrieve('cus_ICiIH7WIPI4Gr1');

// Per request
// \Stripe\Stripe::setApiKey('sk_test...');
//
// echo \Stripe\Customer::retrieve('cus_ICiIH7WIPI4Gr1', [
//   'api_key' => 'sk_test...'
// ]);

// With connect
// \Stripe\Stripe::setApiKey('sk_test...');
//
// echo \Stripe\Customer::retrieve('cus_HDfWzCQ6UEVtfu', ['stripe_account' => 'acct_1Ey3h1BqeQ4DKpna']);


// Client services
// Globally set API key
// $stripe = new \Stripe\StripeClient('sk_test...');
// echo $stripe->customers->retrieve('cus_ICiIH7WIPI4Gr1');

// Per request
// echo $stripe->customers->retrieve(
//   'cus_ICiIH7WIPI4Gr1',
//   [],
//   ['api_key' => 'sk_test...']
// );

// With connect
$stripe = new \Stripe\StripeClient('sk_test...');

echo $stripe->customers->retrieve('cus_HDfWzCQ6UEVtfu', [], ['stripe_account' => 'acct_1Ey3h1BqeQ4DKpna']);
